docs/ServerTraceContext.md

```diff
@@ -0,0 +1,32 @@
+# Server Trace Context
+
+## Rationale
+
+`splunk-rum.js` captures the server trace context from a `Server-Timing` header in the `traceparent` format.  Our
+server-side instrumentation generates this automatically.  This document describes how to recreate this if you want to 
+use an alternate form of instrumentation (i.e., creating this header manually).
+
+## Specification
+
+Please familiarize yourself with these specifications:
+- <https://www.w3.org/TR/server-timing/>
+- <https://www.w3.org/TR/trace-context/#traceparent-header>
+
+We expect a Server-Timing entry with the name `traceparent` where the `desc` field holds the
+corresponding value:
+
+```http
+Server-Timing: traceparent;desc="00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-00"
```

Comment:
Doesn't splunk-rum expect sampled flag to be 01 (https://github.com/signalfx/splunk-browser-otel-js/blob/main/src/servertiming.js#L27)?

Reply:
Excellent catch!